src/utils.ts

```diff
@@ -308,6 +308,51 @@ export function readContract(contractPath: string) {
     };
 }
 
+export function readCairo1Contract(contractPath: string) {
+    const { parse, stringify } = handleJsonWithBigInt(false);
+    const parsedContract = parse(fs.readFileSync(contractPath).toString("ascii")) as ContractClass;
+    return {
+        contract_class_version: parsedContract.contract_class_version,
+        entry_points_by_type: parsedContract.entry_points_by_type,
+        abi: formatSpaces(stringify(parsedContract.abi)),
+        sierra_program: compressProgram(formatSpaces(stringify(parsedContract.sierra_program)))
+    };
+}
+
+export function formatSpaces(json: string) {
```

Comment:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/d71f4425310e0e2255d511386b1347bcbdf3b362 There's a slight difference in `formatSpaces` than `JSON.stringify()` with the optional argument. This will avoid new lines.